src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -69,6 +69,40 @@ protected function doSend(MessageInterface $message): SlackSentMessage
         }
 
         $options = $options?->toArray() ?? [];
+
+        if (isset($options['conversation_ids']) 
+            && is_array($options['conversation_ids']) 
+            && count($options['conversation_ids']) > 1
+        ) {
+            $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/api/conversations.open', [
+                'json' => ['users' => implode(',', $options['conversation_ids'])],
+                'auth_bearer' => $this->accessToken,
+                'headers' => [
+                    'Content-Type' => 'application/json; charset=utf-8',
+                ],
+            ]);
+            
+            try {
+                $statusCode = $response->getStatusCode();
+            } catch (TransportExceptionInterface $e) {
+                throw new TransportException('Could not reach the remote Slack server.', $response, 0, $e);
+            }
+
+            if (200 !== $statusCode) {
+                throw new TransportException(sprintf('Unable to post the Slack message: "%s".', $response->getContent(false)), $response);
+            }
+
+            $result = $response->toArray(false);
+            if (!$result['ok']) {
+                $errors = isset($result['errors']) ? ' ('.implode('|', $result['errors']).')' : '';
+    
+                throw new TransportException(sprintf('Unable to post the Slack message: "%s"%s.', $result['error'], $errors), $response);
+            }
+    
+            $options['channel'] = $result['channel']['id'] ?? null;
```

Comment:
If no channel id is in the response, should that not throw an error ?

Reply:
Well it's an exceptional situation if you've failed to create a conversation with specified users